### DISCUSSION TITLE
Enlarge the Font in 2D Map (OSM)

### DISCUSSION THREAD

**Discussion by DummerJunge** [mytourbook/mytourbook#1166] [Q&A] [answered]
The text in the 2D Map (OSM) is even with glasses unreadable small (for me), do I have the oppurtunity to enlarge the fonts for POI, streets ... thank you

**Comment by wolfgang-ch** [comment]
Can you provide a screenshot which shows the text and which size is small

**Reply by DummerJunge** [reply]
Hello, thank's for paying attention, please find a screenshot attached. As far as I now know the tiles come  from the OSM tile server directly (https://tile.openstreetmap.de/tiles/osmde/{zoom}/{x}/{y}.png) and OSM seems not to support changing the fonts used. I found a hint that mapbox tile server could do this but I have to investigate this further ...
![Screenshot 2023-08-16 212641](https://github.com/mytourbook/mytourbook/assets/7841822/82f51d0f-188a-427f-b26a-859d70dcb1e6)

**Reply by wolfgang-ch** [reply, marked as answer]
The font size is rendered on the tile server and MT is providing the tiles as they can be downloaded but I suggest to use also [other map providers](https://mytourbook.sourceforge.io/mytourbook/index.php/map-provider) like the following.

![image](https://github.com/mytourbook/mytourbook/assets/1283445/72cdb5c8-00e7-458a-80e8-a890b0288bb0)

![image](https://github.com/mytourbook/mytourbook/assets/1283445/605b0559-5eb2-41fb-b24b-07516a04cee0)


It seems that this is a strange area where you are moving, with these markers 😎 

* Hölle
* Höllenklippe
* Hexenkessel
* Blumentopf
* Pferdekopf
* Erdbeerkopf
* Spinne

**Reply by DummerJunge** [reply]
Thanks!
